components/datetime/tests/patterns/tests/time_zones.json

```diff
@@ -2,1100 +2,207 @@
-            "Iso8601": [
-              "Extended",
-              "Required",
-              "Never"
-            ]
-          }
-        ],
-        "expected": ["+01:00"]
-      }
-    ]
+    "expectations": {
+      "z": "GMT+1",
+      "zzzz": "Irish Standard Time",
+
+      "v": "GMT",
+      "vvvv": "Greenwich Mean Time",
+
+      "VVV": "Dublin",
+      "VVVV": "Ireland Time",
+
+      "O": "GMT+1",
+      "OOOO": "GMT+01:00"
+    }
   },
   {
     "locale": "ja",
     "datetime": "2021-01-11T12:00:00.000+09:00[Asia/Tokyo]",
-    "expectations": [
-      {
-        "patterns": [
-          "z",
-          "zz",
-          "zzz"
-        ],
-        "configs": [
-          "SpecificNonLocationShort"
-        ],
-        "expected": ["JST"]
-      },
-      {
-        "patterns": [
-          "zzzz"
-        ],
-        "configs": [
-          "SpecificNonLocationLong"
-        ],
-        "expected": ["日本標準時"]
-      },
-      {
-        "patterns": [
-          "Z",
-          "ZZ",
-          "ZZZ"
-        ],
-        "configs": [
-          {
-            "Iso8601": [
-              "Basic",
-              "Required",
-              "Optional"
-            ]
-          }
-        ],
-        "expected": ["+0900"]
-      },
-      {
-        "patterns": [
-          "O"
-        ],
-        "configs": [
-          "LocalizedOffsetShort"
-        ],
-        "fallback_formats": ["LocalizedOffset"],
-        "expected": ["GMT+9"]
-      },
-      {
-        "patterns": [
-          "OOOO",
-          "ZZZZ"
-        ],
-        "configs": [
-          "LocalizedOffsetLong"
-        ],
-        "expected": ["GMT+09:00"]
-      },
-      {
-        "patterns": [
-          "ZZZZZ"
-        ],
-        "configs": [
-          {
-            "Iso8601": [
-              "UtcExtended",
-              "Required",
-              "Optional"
-            ]
-          }
-        ],
-        "expected": ["+09:00"]
-      },
-      {
-        "patterns": [
-          "v"
-        ],
-        "configs": [
-          "GenericNonLocationShort"
-        ],
-        "expected": ["JST"]
-      },
-      {
-        "patterns": [
-          "vvvv"
-        ],
-        "configs": [
-          "GenericNonLocationLong"
-        ],
-        "expected": ["日本時間"]
-      },
-      {
-        "patterns": [
-          "VVV"
-        ],
-        "configs": [],
-        "expected": ["東京"]
-      },
-      {
-        "patterns": [
-          "VVVV"
-        ],
-        "configs": [
-          "GenericLocation"
-        ],
-        "expected": ["日本時間"]
-      },
-      {
-        "patterns": [
-          "x",
-          "X"
-        ],
-        "configs": [
-          {
-            "Iso8601": [
-              "UtcBasic",
-              "Optional",
-              "Never"
-            ]
-          }
-        ],
-        "expected": ["+09"]
-      },
-      {
-        "patterns": [
-          "xx",
-          "XX",
-          "xxxx",
-          "XXXX"
-        ],
```

Comment:
Seems okay although you lost the currently-unreachable ISO-8601 x/X patterns.

Reply:
Yep, they can easily be unit tested though.